courses/machine_learning/intro_to_tensorflow/3_keras_sequential_api.ipynb

```diff
@@ -0,0 +1,482 @@
+{
```

Comment:
<p>A simpler way to do that would be</p><p> </p><p>RMSE_COLS = ['rmse', 'val_rmse']</p><p>pd.DataFrame(history.history)[RMSE_COLS].plot();</p><p> </p><p>(same thing for the loss plot)</p><p> </p><p>For the above to work, I believe you'd need to do "%matplotlib inline" (a good place would be just after the imports)</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/559/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>